internal/proxy/task_delete.go

```diff
@@ -202,75 +202,78 @@ func repackDeleteMsgByHash(
 	collectionName string,
 	partitionID int64,
 	partitionName string,
-) (map[uint32]*msgstream.DeleteMsg, int64, error) {
+) (map[uint32][]*msgstream.DeleteMsg, int64, error) {
+	maxSize := Params.PulsarCfg.MaxMessageSize.GetAsInt()
 	hashValues := typeutil.HashPK2Channels(primaryKeys, vChannels)
```

Comment:
I found `HashPK2Channels` is very likely to have a conflict since the returned values are reminders of hash value divided by number of channels. Is it OK to use the function here?

Reply:
Old code works success, this pr just split hashed msg. The hash function was OK,  returned values key are hash value mod number of channels.